src/components/Table/TableBody/TableBodyRow/TableBodyRow.jsx

```diff
@@ -419,6 +421,8 @@ const TableBodyRow = ({
             data-offset={offset}
             offset={offset}
             width={matchingColumnMeta && matchingColumnMeta.width}
+            align={align}
+            className={col.isSortable === true ? `data-table-${align}` : `data-table`}
```

Comment:
we should use the classnames module to handle these.

Reply:
Not sure why isSortable is tied with cell alignment. I have removed the tertiary and align defaults to `start`